lightning/src/ln/channel.rs

```diff
@@ -5985,6 +5985,20 @@ pub(super) struct InboundV1Channel<Signer: ChannelSigner> {
 	pub context: ChannelContext<Signer>,
 }
 
+/// A not-yet-accepted inbound (from counterparty) channel using V1
+/// channel establishment. Once accepted, the parameters will be used
+/// to construct a channel.
+pub(super) struct UnacceptedInboundV1Channel {
+	/// The counterparty node id that initiated the channel open.
+	pub counterparty_node_id: PublicKey,
+	/// The original OpenChannel message.
+	pub open_channel_msg: msgs::OpenChannel,
+	/// The outbound SCID alias assigned to the channel when we received
+	/// the OpenChannel request.
+	pub outbound_scid_alias: u64,
+}
```

Comment:
I think we could rename this to `InboundChannelRequest` to just distance it from an association with it being an actual channel at this point.

Also dropping the V1 Prefix to channel as I’ll make the open_channel_msg an enum in the V2 establishment PR so it could hold either an `OpenChannel` or `OpenChannelV2`.

```suggestion
pub(super) struct InboundChannelRequest {
	/// The counterparty node id that initiated the channel open.
	pub counterparty_node_id: PublicKey,
	/// The original OpenChannel message.
	pub open_channel_msg: msgs::OpenChannel,
	/// The outbound SCID alias assigned to the channel when we received
	/// the OpenChannel request.
	pub outbound_scid_alias: u64,
}
```